.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-ci
+repo_token: QnUhpSnr5eFpJLx3fFHRAEqbnIh4Azp9V
```

Comment:
This file should never be committed. Remove it cause it has sensitive information like the `repo_token`.